### ISSUE TITLE
macOS version is not buildable

### ISSUE THREAD

**Issue by nevack** [saket/press#68]
Is macOS version still buildable?
I have encountered a lot of build errors with ReaktiveInterop, ObservableNavigator etc.
Even after bumping some libs (kotlin-coroutines, kotlinx.serialization) and fixing all Swift errors, app is crashing on start with

`ERROR - sqlite3_close(0x7f88d9e3bba0) failed: 5`

**Comment by saket** [comment]
Yeah sorry about that. I was actively building it, but trying to get libgit2 up and running for syncing of notes was proving to be difficult. With the introduction of [Compose for Desktop UI](https://www.jetbrains.com/lp/compose/) I'm no longer sure if I wanna continue down this path. I'm leaning towards trying it out so that I can reuse JVM libraries for git, markdown, etc. Tracking it here: https://github.com/saket/press/issues/63
